eta/core/data.py

```diff
@@ -293,26 +293,35 @@ class AttributeSchema(etal.LabelsSchema):
         type: the fully-qualified name of the Attribute class
         exclusive: whether at most one attribute with this name may appear in
             an AttributeContainer
+        default: an optional default value for the attribute
     '''
 
-    def __init__(self, name, exclusive=False):
+    def __init__(self, name, exclusive=False, default=None):
```

Comment:
So can an `AttributeSchema` have a default value that is not in its schema?

Reply:
Good point- I added validation for that